README.md

```diff
@@ -21,22 +21,23 @@ To use Databricks-managed Unity Catalog with this package, follow the [instructi
 
 #### Prerequisites
 
+- **[Highly recommended]** Use python>=3.10 for accessing all functionalities including function creation and function execution.
```

Comment:
why?

Reply:
Because databricks-sdk==15.1.0 only supports python>=3.10, and we want customers to have a great journey the first time trying out this client, then `serverless` is highly recommended. And to use function creation feature, the user has to use serverless with dbconnect.
I checked DBR 13.3 already supports python>=3.10, and this is only databricks-customers facing, so I think it's fine.

Reply:
got it, thanks for the explanation 👍